src/cow-react/modules/limitOrders/containers/LimitOrdersWidget/index.tsx

```diff
@@ -91,21 +94,26 @@ export function LimitOrdersWidget() {
     field: Field.OUTPUT,
     label: isSellOrder ? 'Your receive at least' : 'You receive exactly',
     currency: outputCurrency,
-    rawAmount: outputCurrencyAmount,
-    viewAmount: outputCurrencyAmount?.toExact() || '',
+    rawAmount: isWrapOrUnwrap ? inputCurrencyAmount : outputCurrencyAmount,
```

Comment:
Maybe in this case it should use `typedValue`?

Reply:
`TypedValue` is a string, `rawAmount` is a CurrencyAmount